src/popup/router/components/AuctionCard.vue

```diff
@@ -49,8 +50,8 @@ export default {
   }
 
   .name {
-    font-weight: bold;
```

Comment:
Looks like in designs we still need bold here 
![image](https://user-images.githubusercontent.com/7098449/140299112-bf2ecf4a-4fd7-457f-8c3f-5a6dfca0f59f.png)

Reply:
Was going to use %face-sans-14-bold here.